gbpservice/nfp/orchestrator/coal/networking/nfp_gbp_network_driver.py

```diff
@@ -45,8 +50,12 @@ def get_port_details(self, token, port_id):
         self.network_handler = openstack_driver.GBPClient(self.config)
         return port_details
 
+    def get_networks(self, token, filters):
+        return self.network_handler.get_policy_target_groups(token,
+                                                             filters=filters)
+
     def set_promiscuos_mode(self, token, port_id):
-        port_id = self.get_port_id(token, port_id)
+        # port_id = self.get_port_id(token, port_id)
```

Comment:
This line is needed.
Since the argument port_id is GBP pt_id, we first need to get corresponding neutron port_id for that pt_id.
So that
super(NFPGBPNetworkDriver, self).set_promiscuos_mode(token, port_id)
will work